lib/features/mailbox_dashboard/presentation/controller/search_controller.dart

```diff
@@ -179,6 +194,19 @@ class SearchController extends BaseController {
     }
   }
 
+  void saveRecentSearch(RecentSearch recentSearch) {
+    consumeState(_saveRecentSearchInteractor.execute(recentSearch));
```

Comment:
`states` come from this interactor will be handle in where?

Reply:
`states ` will not handle (or will be handle in future). This code created on mailbox_dashboard_controller before, I just move it to search controller.